### PR TITLE
Bug 1503845, added more information about collecting token

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1503845

### CODE REVIEW

**Comment by yanpzhan** [comment]
The token info is not on "about" page, it should be on command-line page, which is opened by clicking "Command Line Tools" in help menu.

**Comment by ahardin-rh** [comment]
@yanpzhan Thank you. This is now updated.

File: cli_reference/get_started_cli.adoc [hunk 1]
```diff
@@ -53,6 +53,12 @@ endif::[]
 
 Installation options for the CLI vary depending on your operating system.
 
+To log in using the CLI, collect your token from the web console's *Command
+Line* page, which is accessed from *Command Line Tools* in the *Help* menu. The
+token is hidden, so you must click the *copy to clipboard* button at the end of
+the `oc login` line on the *About* page, then paste the copied contents to show
```
Comment (yanpzhan) [inline cli_reference/get_started_cli.adoc]: This '*About* page' should be updated, too.

**Comment by yanpzhan** [comment]
BTW, there is another way to get the token on web console, on the upper right corner, click the account menu, there is "Copy Login Command", user could get login command with token by clicking it.
But I'm not sure from which version the function was involved, maybe ocp 3.6 or 3.7?

File: cli_reference/get_started_cli.adoc (hunk 1 above)
Comment (ahardin-rh) [reply cli_reference/get_started_cli.adoc]: Thanks! Updated.

**Comment by yanpzhan** [comment]
LGTM now

**Pull request merged**